idea/Hublsoft.Task.Web/Controllers/LoginMonitorController.cs

```diff
@@ -18,10 +19,12 @@ public LoginMonitorController(LoginService service)
 
         public IActionResult Index()
         {
-            var model = _service.GetLoginsForAllUsers();
-            return View(model);
-        }
-
+            var preferJSON = Request.Headers["Accepts"] == "application/json";
+            return preferJSON ? IndexAsJSON() : View();
+        }

```

Comment:
If we are replacing the Razor view, do we need this check? we should remove all code related to Razor implementation if it is not required.